beetsplug/embedart.py

```diff
@@ -60,22 +60,59 @@ def commands(self):
         embed_cmd.parser.add_option(
             u'-f', u'--file', metavar='PATH', help=u'the image file to embed'
         )
+        embed_cmd.parser.add_option(
+            u"-y", u"--yes", action="store_true", help=u"skip confirmation"
+        )
         maxwidth = self.config['maxwidth'].get(int)
         compare_threshold = self.config['compare_threshold'].get(int)
         ifempty = self.config['ifempty'].get(bool)
 
+        def _confirmation(items, opts, fmt, prompt):
+            # Confirm artwork changes to library items.
+                if not opts.yes:
+                    # Prepare confirmation with user.
+                    print_()
+
+                    # Show all the items.
+                    for item in items:
+                        print_(format(item, fmt))
+
+                    # Confirm with user.
+                    if not ui.input_yn(prompt, True):
+                        return False
```

Comment:
There are similarities here, sure, but there is a subtle difference of return returning False instead of nothing. I don't know how to tidy this up any further. If you wouldn't mind doing it so that I can see how it should be done for next time that would be great!